### PR TITLE
Use solid-node-client & rdflib 2.x versions

### PR DESCRIPTION
**Pull request by jeff-zucker**
Replaces solid-auth-cli with solid-node-client.  Uses solid-node-client class interface (need to bind the fetch).  Uses latest rdflib.

### CODE REVIEW

**Review by timbl — APPROVED** [review]
Looks good to me

**Comment by jeff-zucker** [comment]
gitter-solid changes                                                            
--------------------                                                            
* uses 2.x versions of rdflib and solid-node-client                             
* prompts user for all values not found in env variables or parameters          
* supports storing chats in local file system serverless Pod                    
  * creates profile, prefs.ttl, etc. if they dont exist                         
  * stores chat locations in gitterConfiguration.ttl, just as with https        
* currently silences fetchQueue errors (see https://github.com/linkeddata/rdflib.js/issues/461)

File: README.md [hunk 1]
```diff
@@ -55,7 +53,12 @@ In gitter, the concept of a room includes public rooms, private rooms, and priva
...
   ## Places to store the chat
 
-  It is important to make sure that creating a copy of
+  You will be asked if you want to store the chat remotely.  If you answer
+  no, you will be prompted for a local file location (see below). 
+  In either case,  you will then be prompted to give the locations of the 
+  folder for different kinds of chats.
+
+  The separate folders are necessary when using a server-based pod, because it is important to make sure that creating a copy of
   chat data in a solid pod does not give anyone access to is
```
Comment (TallTed) [inline README.md]:
```suggestion
  chat data in a solid pod does not give anyone access to it
```

File: README.md [hunk 2]
```diff
@@ -68,19 +71,25 @@ In gitter, the concept of a room includes public rooms, private rooms, and priva
...
+This must be a full https: or file: URI ending in a slash, which folder on your pod you want gitter chat stored.
 URI for publicChatFolder? https://timbl.com/timbl/Public/Archive/
-
+or
+URI for publicChatFolder? file:///home/jeff/myPod/Public/Archive/
 ```
 
  ## Syntax
 
+ The descriptin of command, room, and optionalPodURI, are shown below as
```
Comment (TallTed) [inline README.md]:
```suggestion
 The description of command, room, and optionalPodURI, are shown below as
```

**Comment by timbl** [comment]
Thanks Jeff!

**Pull request merged**